### DISCUSSION TITLE
Github externship proposal doubt

### DISCUSSION THREAD

**Discussion by devblin** [hoppscotch/hoppscotch#1994] [Q&A]
My github externship profile is still under review, though I have started my proposal draft; so, can I start a proposal disucssion, under externship category to get guidance/suggestions related to my proposal?

**Comment by liyasthomas** [comment]
Hi @devblin, yeah of course. Feel free to ask your doubts here.

**Comment by devblin** [comment]
@liyasthomas , In github externship isn't **hoppscotch** accepting proposal for **hopp-cli** ?

**Reply by liyasthomas** [reply]
Yes, you can submit proposal for any public repository under the Hoppscotch organization.

**Comment by devblin** [comment]
Should I submit my proposal first in github externship website ? Or should I get my proposal reviewed within Hoppscotch's Github externship disucussion and then submit application ?
